kubernetes/katago-server/templates/ingress.yaml

```diff
@@ -22,4 +22,8 @@ spec:
       - path: /
         backend:
           serviceName: django-service
-          servicePort: {{.Values.django.servicePort}}
\ No newline at end of file
+          servicePort: {{.Values.django.servicePort}}
+      - path: /
```

Comment:
That looks false to me. You can't have two same path for the same host.

I will create another host eg `flower.katagotraining.org`

Reply:
You can't have two paths on the same host even if they are different ports?
Flower is on port 5555, right? This is how we had it in docker-compose production.yml too, right?. We have path "/" with port 80 or port 443 go to django, except that if also has the prefix "/media" it goes nginx. And we had path "/" on port 5555 to to flower?

Reply:
I'm almost certain it won't work: https://cloud.google.com/kubernetes-engine/docs/concepts/ingress

The ingress itself is a loadblancer listenning on only 80 and 443 (like traefik by default)

![image](https://user-images.githubusercontent.com/13785185/98447536-cb1b4280-2125-11eb-9228-a6352b97a358.png)

to be able to reach `whatever.url.com:5555` we do not only need the internal pod to listen on 5555 but also the ingress.
On traefik we opened a new endpoint for this.

On GCP i don't know how to do, with defult ingress controller. (with another ingress controller, you can do this for instance https://kubernetes.github.io/ingress-nginx/user-guide/exposing-tcp-udp-services/)

My point isnon standard port like 5555 is non standard so we should prefer having two hosts

Reply:
Also we have not configured https yet: here is how to do https://cloud.google.com/kubernetes-engine/docs/how-to/managed-certs

Reply:
Maybe we should just use nginx ingress controller then?

Reply:
not sure we really need to expose a custom port

we can have

`katagotraining.com -> django`
`flower.katagotraining.com ->flower`

that would be more clean for sure